src/v2/Components/FlashBanner/__stories__/FlashBanner.story.tsx

```diff
@@ -5,6 +5,11 @@ import { FlashBanner } from "v2/Components/FlashBanner"
 storiesOf("Components/FlashBanner", module).add(
   "Successful Confirmation Message",
   () => {
-    return <FlashBanner messageCode="confirmed" />
+    return (
+      <FlashBanner
+        contentCode="confirmed"
+        me={{ canRequestEmailConfirmation: false } as any}
+      />
+    )
```

Comment:
It would be nice if there were a series of stories here representing the different content codes. Don't need to worry about relay or query params and can just use props, but something so future devs can see at a glance that there are a few different states.